bikeshed/boilerplate.py

```diff
@@ -12,8 +12,8 @@
 from .translate import _
 
 if t.TYPE_CHECKING:
-    MetadataT: t.TypeAlias = t.Mapping[str, t.Sequence[MetadataValueT]]
     MetadataValueT: t.TypeAlias = str | t.NodesT | None
+    MetadataT: t.TypeAlias = t.Mapping[str, t.Sequence[MetadataValueT]]
```

Comment:
I swapped these two lines, since I noticed in the Problems tab in vscode that it said MetadataValueT was undefined.  Not sure this is correct, but at least that problem goes away :)  Let me know what to do otherwise.

Reply:
Sigh, annoying that VSCode apparently isn't recognizing that the code is intentionally hidden from the Python runtime. The order doesn't matter for mypy and Python doesn't execute that code (since it's under an `if` that's always false at runtime), so there's no need for you to do anything here.